### PR TITLE
Formatting fix

### PR DESCRIPTION
**Pull request by gonnet**
Formatting fix in documentation of `zeros_like` in backend.py.

### CODE REVIEW

File: tensorflow/python/keras/backend.py [hunk 1]
```diff
@@ -1679,10 +1679,10 @@ def zeros_like(x, dtype=None, name=None):
   Example:
 
 
-  from tensorflow.keras import backend as K
-  kvar = K.variable(np.random.random((2,3)))
-  kvar_zeros = K.zeros_like(kvar)
-  K.eval(kvar_zeros)
+  >>> from tensorflow.keras import backend as K
```
Comment (fchollet) [inline tensorflow/python/keras/backend.py:1682]: Please instead use the ``` code block formatting syntax since these lines are not meant to be executed as part of our "doctest" suite (testable code examples in docstrings).

**Review by fchollet — CHANGES_REQUESTED** [review]
Thank you for the PR

File: tensorflow/python/keras/backend.py (hunk 1 above)
Comment (gonnet) [reply tensorflow/python/keras/backend.py]: Done!

**Review by fchollet — APPROVED** [review]
LGTM, thanks

**Comment by gonnet** [comment]
Is there any way to re-run failing tests?

**Comment by mihaimaruseac** [comment]
Please use testable docstrings instead of inline code blocks. See https://www.tensorflow.org/community/contribute/docs_ref

**Comment by gonnet** [comment]
> Please use testable docstrings instead of inline code blocks. See https://www.tensorflow.org/community/contribute/docs_ref

Now I'm a bit confused... I had done that originally, but @fchollet asked me to use code block formatting instead, since "these lines are not meant to be executed as part of our "doctest" suite".

So should these lines be a doctest or not?

Cheers, Pedro

**Review by mihaimaruseac — APPROVED** [review]

**Comment by mihaimaruseac** [comment]
You are right, my bad

**Comment by mihaimaruseac** [comment]
Adding the kokoro label retriggers the tests, but it has to be done by a googler.

Right now the PR is waiting import into Google (`import/copybara`) and internal review. The MacOS CPU failed build will not block merge (since it is unrelated)
